### PR TITLE
Command Pattern

### PR DESCRIPTION
**Pull request by sasalx**
Added Command Pattern.

### CODE REVIEW

File: Unal/Design Patterns/Composite Pattern/LAB3/main.cpp [hunk 1]
```diff
@@ -1,8 +1,3 @@
-//============================================================================
-// Name        : CompositePattern.cpp
-// Unal Muslu 20150602025 LAB3
-//============================================================================
-
```
Comment (kondanta) [inline Unal/Design Patterns/Composite Pattern/LAB3/main.cpp]: Why?

File: Unal/Design Patterns/Command Pattern/LAB4/main.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+int main(){
+    srand(static_cast<unsigned int>(time(nullptr))); // For random seed gen.
+
+    EventHandler *event = nullptr;
+    auto *user = new User();
+    auto *mDriver = new MouseDriver();
+
+    event = new MouseEventHandler(mDriver);
+    user->Work(event);
+
```
Comment (kondanta) [inline Unal/Design Patterns/Command Pattern/LAB4/main.cpp]: There are memory leaks

File: Unal/Design Patterns/Command Pattern/LAB4/main.cpp [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+#include <vector>
+#include <ctime>
+#include <fstream>
+
+using namespace std;
+
+// EventHandler (Command)
+
+class EventHandler {
+public:
```
Comment (kondanta) [inline Unal/Design Patterns/Command Pattern/LAB4/main.cpp]: You suppose to add dtor for removing derived class objects

Comment (sasalx) [reply Unal/Design Patterns/Command Pattern/LAB4/main.cpp]: Fixed.

**Pull request merged**